### ISSUE TITLE
Theme problems

### ISSUE THREAD

**Issue by Corantin** [1Hive/gardens-v2#330]

**Comment by Mati0x** [comment]
@Corantin I dont get the issue:
 - We dont have secondary buttons
 - The disabled button and a cancel button are like this: 

![Image](https://github.com/user-attachments/assets/d48cdfc6-70e4-4711-bbd5-43e660bd832c)

**Comment by Corantin** [comment]
Yeah at this time I forgot we had the Button Component but there is still some problems with it let me discuss it in a voice :p
